release/release_tests.yaml

```diff
@@ -4122,7 +4122,7 @@
   run:
     timeout: 3600
     long_running: false
-    script: python run_llm_serve_release_tests.py --serve-config-file configs/serve_llama_3dot2_1b_no_accelerator.yaml
+    script: python run_llm_serve_release_tests.py --serve-config-file configs/serve_llama_3dot2_1b_no_accelerator.yaml --run-perf-profiler false
```

Comment:
can you add a comment above `llm_serve_llama_3dot1_8B_tp_2` and `llm_serve_llama_3dot1_8B_quantized_tp_1` to indicate that these two tests also run the performance benchmarks against vllm?